db/rdbms/src/main/resources/mapper/ProcessInstanceMapper.xml

```diff
@@ -41,32 +41,30 @@
 
   <!-- default search statement for databases supporting LIMIT/OFFSET-->
   <select id="search" parameterType="io.camunda.db.rdbms.read.domain.ProcessInstanceDbQuery" resultMap="io.camunda.db.rdbms.sql.ProcessInstanceMapper.searchResultMap">
+    SELECT * FROM (
     SELECT
-      pi.PROCESS_INSTANCE_KEY,
-      pi.PROCESS_DEFINITION_ID,
-      pi.PROCESS_DEFINITION_KEY,
-      pi.STATE,
-      pi.START_DATE,
-      pi.END_DATE,
-      pi.TENANT_ID,
-      NULL as TREE_PATH,
-      NULL as INCIDENT,
-      pi.PARENT_PROCESS_INSTANCE_KEY,
-      pi.PARENT_ELEMENT_INSTANCE_KEY,
-      pi.ELEMENT_ID,
-      pi.VERSION,
-      pd.NAME AS PROCESS_DEFINITION_NAME,
-      pd.VERSION_TAG AS PROCESS_DEFINITION_VERSION_TAG
+    pi.PROCESS_INSTANCE_KEY,
+    pi.PROCESS_DEFINITION_ID,
+    pi.PROCESS_DEFINITION_KEY,
+    pi.STATE,
+    pi.START_DATE,
+    pi.END_DATE,
+    pi.TENANT_ID,
+    NULL as TREE_PATH,
+    NULL as INCIDENT,
+    pi.PARENT_PROCESS_INSTANCE_KEY,
+    pi.PARENT_ELEMENT_INSTANCE_KEY,
+    pi.ELEMENT_ID,
+    pi.VERSION PROCESS_DEFINITION_VERSION,
+    pd.NAME AS PROCESS_DEFINITION_NAME,
+    pd.VERSION_TAG AS PROCESS_DEFINITION_VERSION_TAG
```

Comment:
❓Can't we just use the regular column names for these too?

Reply:
Technically yes. But since the values come from a different table I always like to prefix the columns to make it more explicit and we do not mix something up.

It's also just inside the mapper, since the OR-mapping also happens inside the mapper file.